### ISSUE TITLE
Request: dump mock file system for easy inspection

### ISSUE THREAD

**Issue by abelnation** [tschaub/mock-fs#90]
a method like `mockFs.dump()` would be nice for debugging/inspecting

**Comment by jprichardson** [comment]
You can do this by getting the `binding`. 

Example:

```js
var mockfs = require('mock-fs')
var fs = mockfs.fs()
var binding = fs.__get__('binding')
console.dir(binding._system._root)
```

**Comment by ciaranj** [comment]
As far as I can tell this only works for mockfs.fs, rather than mockfs(). Is that intentional?

**Comment by tschaub** [comment]
This was fixed by @ciaranj in #194 and is included in the `mock-fs@4` release.
